src/Operators/vorticity_operators.jl

```diff
@@ -18,7 +18,7 @@
 
  The vertical circulation associated with horizontal velocities ``u`` and ``v``.
  """
-@inline function Γᶠᶠᶜ(i, j, k, grid::OrthogonalSphericalShellGrid, u, v)
+@inline function Γᶠᶠᶜ(i, j, k, grid::OrthogonalSphericalShellGrid{<:Any, <:Any, <:Any, <:Any, <:Any, <:Any, <:Any, <:ConformalMapping, <:Any}, u, v)
```

Comment:
I suggest you define, e.g.,

```Julia
const ConformalCubedSpherePanel = OrthogonalSphericalShellGrid{<:Any, <:Any, <:Any, <:Any, <:Any, <:Any, <:Any, <:ConformalMapping}
```

and then dispatch over the `ConformalCubedSpherePanel` type.

Reply:
Yes, it does make the code clearer. Thanks @navidcy!